zuul.d/edpm_multinode.yaml

```diff
@@ -445,6 +445,33 @@
     parent: podified-multinode-edpm-deployment-crc-bootstrap-staging
     vars:
       cifmw_enable_virtual_baremetal_support: true
+      cifmw_config_nmstate: true
+      cifmw_ci_nmstate_instance_config:
+        controller:
+          interfaces:
+            - name: "{{ cifmw_networking_env_definition.instances['controller'].networks['baremetal'].interface_name | default(omit) }}"
```

Comment:
Based on the error in CI, I guess the `default(omit)` ends up being used as the name, either here or on L464?

Reply:
Fixed, we weren't loading the cifmw_networking_env_definition in the deploy-edpm playbook